src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/SignedXml.cs

```diff
@@ -409,7 +409,7 @@ public void ComputeSignature()
             HashAlgorithm hashAlg = signatureDescription.CreateDigest();
             if (hashAlg == null)
                 throw new CryptographicException(SR.Cryptography_Xml_CreateHashAlgorithmFailed);
-            byte[] hashvalue = GetC14NDigest(hashAlg);
+            _ = GetC14NDigest(hashAlg);
```

Comment:
The gist of this one is that `hashAlg` carries state and `GetC14NDigest` is being used to update the hash, as well as cache it. The return value isn't used but the call needs to be made still to update the hash, which is used a little below here in the signature formatter.

The whole `GetC14NDigest` could probably be refactored to be more clear, but there are a lot of dragons in the file and I'd likely just break something.